GetStream/stream-chat-android
# Fix broken Docusaurus links

### 🎯 Goal

This PR fixes broken links in our Docusaurus documentation. Due to the bug reported [here](https://github.com/facebook/docusaurus/issues/5659) had to remove links from the commented sections. 

### 🧪 Testing

1. Navigate to the project root directory
2. Execute `npx stream-chat-docusaurus -i -s`
3. Observe that there are no warnings in console

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
